### PR TITLE
ROX-14172: Fix Configmanagement UI test.

### PR DESCRIPTION
**Pull request by dhaus67**
## Description

With the change of #4031 , the cluster permission evaluator was changed.

Previously, it was sufficient enough to only specify `Cluster role == true` for cluster role bindings, however this has changed.

A cluster role binding has `Cluster role == true && Namespace == ""`.

The query that is currently being used in the cluster permission evaluator (using `AddNullField`) does not match. The reason for this is that there are two namespace fields within the role binding proto message: one for the binding, and one for the subject. Since the subject's namespace is never empty, this query didn't work properly.

This lead to a failing UI test, since the cluster admin roles were not shown correctly.

By fixing the query as well as deleting unused code + making sure that other places respect the condition of `cluster role == true && namespace == ""`, this should be fixed.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

- see CI.

### CODE REVIEW

**Comment by dhaus67** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4269** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/4269" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/4269?utm_source=stack-comment).

File: pkg/k8srbac/evaluator.go [hunk 1]
```diff
@@ -45,39 +44,11 @@ func MakeClusterEvaluator(roles []*storage.K8SRole, bindings []*storage.K8SRoleB
 	// Collect cluster role bindings.
 	var clusterBindings []*storage.K8SRoleBinding
 	for _, binding := range bindings {
-		if binding.GetClusterRole() {
+		// A binding is only binding cluster-wide if cluster role is true _and_ namespace is empty. Namespaced
+		// bindings may bind to cluster roles as well.
+		if binding.GetClusterRole() && binding.GetNamespace() == "" {
```
Comment (janisz) [inline pkg/k8srbac/evaluator.go:49]: This condition appears 3 times how about extracting it to well named function in `pkg`?

**Review by janisz — APPROVED** [review]

**Pull request merged**